packages/react/.storybook/preview.js

```diff
@@ -17,7 +17,7 @@ export const parameters = {
 };
 
 const medplum = new MockClient();
-medplum.requestSchema('Patient');
+medplum.get('/');
```

Comment:
Calling `.get` to call mockFetch() which will initialize the mock repo to resolve race condition of Mock Repo initiated in MockClient.

https://github.com/medplum/medplum/blob/8d381d131c4f32a0fabd301b5fd227960e2d2928/packages/mock/src/client.ts#L183. Any subsequent graphql calls (like medplum.requestSchema) will no longer throw exception.